code/modules/mob/living/simple_animal/abnormality/_tools/zayin/wishwell.dm

```diff
@@ -0,0 +1,250 @@
+/obj/structure/toolabnormality/wishwell
+	name = "wishing well"
+	desc = "A well constructed of stone and wood. From where does it draw water?"
+	icon_state = "wishwell"
+	var/output
+	var/gacha
+	var/gift
+
+//loot lists
+	var/list/superEGO = list( //do NOT put this in the loot lists ever. SuperEGO is for inputs only so people can throw away twilight for no good reason.
+		/obj/item/ego_weapon/paradise,
+		/obj/item/clothing/suit/armor/ego_gear/twilight,
+		/obj/item/ego_weapon/twilight
+		)
+	var/list/alephitem = list(//less junk items at higher risk levels
+		/obj/item/clothing/suit/armor/ego_gear/praetorian,
+		/obj/item/toy/plush/mosb,
+		/obj/item/toy/plush/melt
+		)
+	var/list/wawitem = list(
+		/obj/item/toy/plush/qoh,
+		/obj/item/toy/plush/bigbird,
+		/obj/item/toy/plush/rabbit,
+		/obj/item/grenade/spawnergrenade/shrimp/super,
+		/obj/item/ego_weapon/flower_waltz
+		)
+	var/list/heitem = list(
+		/obj/item/gun/ego_gun/sodashotty,
+		/obj/item/gun/ego_gun/sodarifle,
+		/obj/item/gun/ego_gun/sodasmg,
+		/obj/item/clothing/suit/armor/ego_gear/lutemis,
+		/obj/item/grenade/spawnergrenade/shrimp,
+		/obj/item/clothing/neck/beads,
+		/obj/item/clothing/glasses/sunglasses/reagent,
+		/obj/item/clothing/suit/hawaiian
+		)
+	var/list/tethitem = list(
+		/obj/item/clothing/suit/armor/ego_gear/training,
+		/obj/item/ego_weapon/training,
+		/obj/item/clothing/suit/armor/ego_gear/rookie,
+		/obj/item/clothing/suit/armor/ego_gear/fledgling,
+		/obj/item/clothing/suit/armor/ego_gear/apprentice,
+		/obj/item/clothing/suit/armor/ego_gear/freshman,
+		/obj/item/clothing/under/suit/lobotomy/extraction/arbiter,
+		/obj/item/clothing/under/suit/lobotomy/architecture,
+		/obj/item/clothing/under/suit/lobotomy/rabbit,
+		/obj/item/clothing/under/color/rainbow
+		)
+	var/list/zayinitem = list(
+		/obj/item/reagent_containers/food/drinks/soda_cans/wellcheers_red,
+		/obj/item/reagent_containers/food/drinks/soda_cans/wellcheers_white,
+		/obj/item/food/bread/bongbread,
+		/obj/item/clothing/mask/cigarette/cigar/havana
+		)
+	var/list/normalitem = list(
+		/obj/item/reagent_containers/hypospray/medipen/salacid,
+		/obj/item/reagent_containers/hypospray/medipen/mental,
+		/obj/item/ego_weapon/tutorial,
+		/obj/item/ego_weapon/tutorial/white,
+		/obj/item/ego_weapon/tutorial/black,
+		/obj/item/ego_weapon/tutorial/pale,
```

Comment:
I would say "It's not smart to give players pale but honestly this is 7 pale IIRC it should be ok